### PR TITLE
add ci

### PR DESCRIPTION
**Pull request by hsouf**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - uses: actions/checkout@v3
+
+      - name: Install Protostar
+        shell: bash
+        run: |
+          curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash
+      - name: Build contracts
+        shell: bash
+        run: |
+          exec bash  && source ~/.bashrc && protostar build
```
Comment (achab) [inline .github/workflows/tests.yml:25]: how is `.bashrc` created?

Comment (hsouf) [reply .github/workflows/tests.yml:25]: after running $curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash

Comment (hsouf) [reply .github/workflows/tests.yml:25]: lint is handled by protostar that's why I didn't add a check in the workflow...

Comment (achab) [reply .github/workflows/tests.yml:25]: cool, thanks for clarifying!

Comment (achab) [reply .github/workflows/tests.yml:25]: about lint, you can add pre-commit and pre-push to our repo in this PR, see url -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash

Comment (achab) [reply .github/workflows/tests.yml:25]: will be amazing to have ci + hooks from the beginning, thanks for that

**Pull request closed without merging**